client/wildcard/src/components/Icon/Icon.tsx

```diff
@@ -29,6 +29,11 @@ interface BaseIconProps extends SVGProps<SVGSVGElement> {
      * Defaults to `true`.
      */
     inline?: boolean
+    /**
+     * Brand icon variation, added as of 11/11/22
+     * (https://handbook.sourcegraph.com/departments/engineering/design/brand_guidelines/)
+     */
+    violetBg?: boolean
```

Comment:
Suggestion for a more semantic naming. Also, no date is required, we have Git history.

```suggestion
    /**
     * Brand icon variation with violet background
     * 
     * More info: https://handbook.sourcegraph.com/departments/engineering/design/brand_guidelines/
     */
    branded?: boolean
```